Topics/Tech_Stacks/Tiptap/Tiptap.md

```diff
@@ -269,8 +269,63 @@ npm start
 
 
 ## Example with Markdown Editor in React
-_Include gifs, code snippets, explainations_
 
+By default, Tiptap doesn't support input from or output to Markdown files.[^4] However, there is a community-developed npm package called `tiptap-markdown` that extends Tiptap to enable this. We'll go through the installation of this package and the creation of a simple editor that uses it.
+
+**1. Install `tiptap-markdown`:**
+We'll continue with the React project we created above. Install the [`tiptap-markdown`](https://www.npmjs.com/package/tiptap-markdown) package using npm:
+```
+npm install tiptap-markdown
+```
+
+**2. Import the package:**
+Make a copy of Tiptap.jsx called Markdown.jsx. Add this import to the top of the file:
+```
+import { Markdown } from 'tiptap-markdown';
+```
+
+**3. Edit the editor:**
+In Markdown.jsx, add `Markdown` to the list of extensions used by the Tiptap editor. Since the input file format has changed from HTML to Markdown, we'll change the initial content as well:
+````
+const extensions = [
+  [...keep the existing extensions...]
+  Markdown,
+]
+
+const content = `
+## Hi there,
+Isn’t that great? And all of that is editable. But wait, there’s more. Let’s try a code block:
```

Comment:
change to something like "Hey this is now a markdown, the output now is in .md form"